Adding barley, maize, tomato

jmattock5 opened the pull request:
Added barley, maize and tomato rhizosphere to the schema 🍅

<!--
# ebi-metagenomics/biosiftr pull request

Many thanks for contributing to ebi-metagenomics/biosiftr!

Please fill in the appropriate checklist below (delete whatever is not relevant).
These are the most common things requested on pull requests (PRs).

Remember that PRs should be made against the dev branch, unless you're preparing a pipeline release.

Learn more about contributing: [CONTRIBUTING.md](https://github.com/ebi-metagenomics/biosiftr/tree/master/.github/CONTRIBUTING.md)
-->

jmattock5 committed:
Update nextflow_schema.json

Added barley, maize and tomato rhizosphere to schema

mberacochea reviewed (commented):
Thank you very much @jmattock5!
The tests are not happy, I'll try to take a look tomorrow

Ales-ibt reviewed (changes requested):
Thank you for this. It looks good to me. Do you mind to add the to the README as well? In the bwamem2 db size list and in the table of Availabe biomes

jmattock5 committed:
Update README.md

Added the bwa file sizes and updated the available biomes table

mberacochea committed:
CI only using the latest version of NF (to save some credits)

Also removed the pipeline reelase announcment gh action

mberacochea committed:
Run nf-test CI using NF version 25.04.7

The latest edge version introduced a change that nf-test is not handling.

https://github.com/askimed/nf-test/issues/320

mberacochea reviewed (approved):
:shipit:
